client/src/components/createCredentialDialog/createCredentialDialog.js

```diff
@@ -591,24 +536,22 @@ class CreateCredentialDialog extends React.Component {
 }
 
 CreateCredentialDialog.propTypes = {
-  show: PropTypes.bool,
-  editing: PropTypes.bool,
-
+  addCredential: PropTypes.func,
   getCredentials: PropTypes.func,
-  credentials: PropTypes.array,
```

Comment:
Indeed, looks like it dropped a few lines to [line 554](https://github.com/cdcabrera/quipucords/blob/6d011e25f91a8f1ec976944b5c30daccb1cdb761/client/src/components/createCredentialDialog/createCredentialDialog.js#L554) ... grouped them by how they're merged through ```mapStateToProps```. 

Taking a look at some of the validation pieces across the app. I think credentials in this component is used to confirm a name update doesn't exist against "listed" credentials. If paging goes into place feels like it may play havok with the check

Reply:
arghh.... you're right. Maybe then we remove the check and let it fail and pray we get back a decent error message.